api/read.yaml

```diff
@@ -127,10 +127,15 @@ components:
           description: The mode of read operation. Currently only asynchronus operation is supported.
           enum: [async]
           example: async
-        since:
+        sinceTimestamp:
           type: string
           description: The timestamp from which to read data. If omitted, we will read all data.
           example: "2024-07-01T18:22:30.323771761Z"
+        since:
```

Comment:
no need to keep since field, since we haven't released this API yet, no need to keep backwards compatibility.

Reply:
made the change, thanks!